docs/bundles/plain.md

```diff
@@ -2,43 +2,147 @@
 
 ## Overview
 
-This document is meant to define the `plain` bundle format as a reference for those publishing `plain` bundles
-for use with RukPak. For more information on the concept of a bundle, click [here](https://github.com/operator-framework/rukpak#bundle).
+This document is meant to define the plain bundle format as a reference for those publishing plain bundles for use with
+RukPak. A bundle is a collection of Kubernetes resources that are packaged together for the purposes of installing onto
+a Kubernetes cluster. A bundle can be unpacked onto a running cluster, where controllers can then create the underlying
+content embedded in the bundle. The bundle can be used as the underlying `spec.source` for
```

Comment:
proposing to rephrase: 
" A bundle can be unpacked onto a running cluster, where controllers can then create the underlying
content embedded in the bundle. The bundle can be used as the underlying `spec.source` for a [`Bundle`](https://github.com/operator-framework/rukpak#bundle) resource."

to

Users can specify through  a [`Bundle`](https://github.com/operator-framework/rukpak#bundle) or a [`BundleDeployment`](https://github.com/operator-framework/rukpak#bundledeployment)  a resource bundle that is to be installed. Controllers, called provisioners, consume the bundle referenced in the`spec.source.ref` field of the Bundle resource or the `spec.template.spec.source.ref` of the BundleDeployment and apply the manifests embedded in it.

Reply:
https://github.com/operator-framework/rukpak/pull/642/commits/902da450e601fa3c1b012ca90b21745fb1ecaa25